lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx

```diff
@@ -52,12 +56,17 @@ class HelpPanel extends React.Component {
         <div
           css={`
             svg {
-              width: 15px;
-              height: 15px;
+              width: 16px;
+              height: 16px;
+              fill: #404855;
```

Comment:
Have a look at how I'm thinking about dealing with this (both the width and fill requiring custom css everywhere) and let me know what you think?:
https://github.com/TerriaJS/terriajs/pull/4162/files?file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.md&file-filters%5B%5D=.scss&file-filters%5B%5D=.svg&file-filters%5B%5D=.tsx#diff-2975e3acc006dfdbf49352a33d1e25f3R84

Reply:
Definitely a cleaner solution, will start doing it this way 😄